docs/topics/mta-rn-upgrade-ui.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-rn-upgrade-ui_{context}"]
+= Setting the parameter in the {WebName}
+
+You can set the value of `rwx_supported` when you install the {ProductShortName} {WebName}.
+
+.Procedure
+
+. Follow steps 1-8 of the installation instructions in the link:{ProductDocUserInterfaceGuideURL}/index#mta-6-installing-web-console-on-openshift_user-interface-guide[user interface guide].
```

Comment:
Hi @RichardHoch, I think it would be better to guide the user to follow steps 1-**7**, so once the user opens the Create Tackle window, he will switch to the Yaml form.
And you already mentioned in the next step "make any needed changes to the CR settings" (actually equals to step 8 in the installation instructions)